src/EnergyPlus/EvaporativeCoolers.cc

```diff
@@ -4031,6 +4048,14 @@ namespace EvaporativeCoolers {
                                 "System",
                                 "Average",
                                 ZoneEvapUnit(UnitLoop).Name);
+            if (ZoneEvapUnit(UnitLoop).ControlSchemeType != ZoneCoolingLoadVariableSpeedFan) {
+                SetupOutputVariable("Zone Evaporative Cooler Unit Part Load Ratio",
+                                    OutputProcessor::Unit::None,
+                                    ZoneEvapUnit(UnitLoop).UnitPartLoadRatio,
+                                    "System",
+                                    "Average",
+                                    ZoneEvapUnit(UnitLoop).Name);
+            }
```

Comment:
Can you clarify why this variable isn't relevant to a ZoneCoolingLoadVariableSpeedFan configuration?

Reply:
The variable speed fan modulates the supply air flow to meet the cooling load, so does not need to cycle.

Reply:
Yeah OK makes good sense :smile: